components/Button/index.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export const Button = ({ setProduct, produkt, productToBuy }) => {
+  return (
+    <StyledButton
+      onClick={() => {
+        setProduct(produkt);
+        console.log(productToBuy);
+      }}
+    >
+      Buy
+    </StyledButton>
+  );
+};
+
+const StyledButton = styled.button`
```

Comment:
looks good to me, just of contiunity it might be an idea to get the StyledNutton befroew the Button